### PR TITLE
Backend implementation for work page

### PR DESCRIPTION
**Pull request by hitochan777**
- Backend (GraphQL)
  - [x] List work of users
  - [x] Create work
  - [x] Update work
  - [x] ~~Delete work~~ (Not doing in this PR)

### CODE REVIEW

**Review by willjw3 — APPROVED** [review]
I'm ok with merging, but what about the circle ci checks?

**Comment by hitochan777** [comment]
> I'm ok with merging, but what about the circle ci checks?

CircleCI check now passes

File: .circleci/config.yml [hunk 1]
```diff
@@ -16,9 +16,7 @@ jobs:
 
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "package-lock.json" }}
-            # fallback to using the latest cache if no exact match is found
```
Comment (hitochan777) [inline .circleci/config.yml:20]: Fallbacking to the latest cache when there is no cache with exact match was the cause of circleci check failure,

File: .circleci/config.yml [hunk 2]
```diff
@@ -28,7 +26,7 @@ jobs:
       - save_cache:
           paths:
             - node_modules
-          key: v1-dependencies-{{ checksum "package-lock.json" }}
+          key: v2-dependencies-{{ checksum "package-lock.json" }}
```
Comment (hitochan777) [inline .circleci/config.yml:29]: Cleared circleci cache by incrementing cache version

**Pull request merged**